docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
       - ASPNETCORE_URLS=http://+:80
     ports:
       - "80:80"
+      - "433:433"
```

Comment:
won't work without updating ASPNETCORE_URLS

I removed this because of development certificate issues, did you add it back or is it just leftover

Reply:
Uhhh I never messed with this so no idea where this came from ;;;

Reply:
Could you remove it please